docs/faq.md

```diff
@@ -38,6 +38,113 @@ targets:
           compiler: dart2js
 ```
 
+## How can I build with multiple configurations?
+
+The build system supports two types of builds, "dev" and "release". By default
+with `build_runner` the "dev" version is built by regardless of the command
+used, build in release mode by passing the `--release` flag. With `webdev` the
+default mode for the `serve` command is dev, and the default mode for the
+`build` command is release. The `build` command can use dev mode with the
+`--no-release` flag.
+
+Options can be configured per mode, and they are [merged by
+key](#how-is-the-configuration-for-a-builder-resolved) with the defaults
```

Comment:
@kevmoo - do you know if there is any way to make this shorter and less likely to break if we reword the header?

Reply:
I don't sadly – is there a standard way to put anchors in markdown?

Reply:
Some other markdown renderers support things like `# Really long header that might not be stable {#stable-short-anchor}`

Github doesn't seem to and I can't find any alternative so far. Oh well.